### ISSUE TITLE
Cookie* missing from UWP

### ISSUE THREAD

**Issue by danmosemsft** [dotnet/corefx#17569]
The problem is this, which needs to be resolved now that we need these types in UWP (they're part of standard)

This is the only API that's actually missing from the System.Net namespace (functionality is another thing)

```
// The uap #define is used in some source files to indicate we are compiling classes
// directly into the UWP System.Net.Http.dll implementation assembly in order to use internal class
// methods. Internal methods are needed in order to map cookie response headers from the WinRT Windows.Web.Http APIs.
// Windows.Web.Http is used underneath the System.Net.Http classes on .NET Native. Having other similarly
// named classes would normally conflict with the public System.Net namespace classes.
// So, we need to move the classes to a different namespace. Those classes are never
// exposed up to user code so there isn't a problem.  In the future, we might expose some of the internal methods
// as new public APIs and then we can remove this duplicate source code inclusion in the binaries.
#if uap
namespace System.Net.Internal
#else
namespace System.Net
#endif
{
```

![image](https://cloud.githubusercontent.com/assets/6385855/24376412/a78a26de-12f0-11e7-81b5-af6b29643b75.png)

**Comment by danmosemsft** [comment]
They are exposed in the ref, which is shared with all configurations, but it has no backing impl in the correct namespace

**Comment by danmosemsft** [comment]
@davidsh who doubtless is famliar with this.

**Comment by danmosemsft** [comment]
Or @CIPop

**Comment by tijoytom** [comment]
Fixed by #17835
